CHANGES/1703.bugfix

```diff
@@ -0,0 +1,3 @@
+Fixed an issue that triggered a `NoneType` exception during a GET request when
+a `distribution` was created with only the `repository_version` (without specifying
+a `repository`).
```

Comment:
```suggestion
Fixed an issue causing an HTTP 500 error when a GET request for a non-existing blob was made to a distribution with only a `repository_version` set.
```
How about this? It does not talk about Python types. Users do not care about them.